### PR TITLE
fix error message of json group by

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：
#4202

## Problem Summary(Required) ：
Fix error message of JSON group by/order by.

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/main/java/com/starrocks/catalog/Type.java [hunk 1]
```diff
@@ -683,8 +683,7 @@ public boolean supportBloomFilter() {
     }
 
     public static final String OnlyMetricTypeErrorMsg =
-            "StarRocks hll and bitmap column must use with specific function, and don't support filter, group by and order by, " +
-                    "please run 'help hll' or 'help bitmap' in your mysql client.";
+            "Type hll, bitmap, json not support aggregation/group by/order by/union";
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/catalog/Type.java:686]: should add percentile

Comment (mofeiatwork) [reply fe/fe-core/src/main/java/com/starrocks/catalog/Type.java]: done

**Comment by mofeiatwork** [comment]
run starrocks_fe_unittest

**Comment by mofeiatwork** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 2 / 3 (66.67%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/Type.java|2|3|66.67%|

**Review by kangkaisen — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Pull request merged**